OrbitClientGgp/ClientGgp.cpp

```diff
@@ -164,6 +180,43 @@ bool ClientGgp::InitCapture() {
   return true;
 }
 
+void ClientGgp::InformUsedSelectedCaptureFunctions() {
+  if (capture_functions_used_.size() != options_.capture_functions.size()) {
+    for (std::string selected_function : options_.capture_functions) {
+      if (!capture_functions_used_.contains(selected_function)) {
+        ERROR("Function matching %s not found; will not be hooked in the capture",
+              selected_function);
+      }
+    }
+  } else {
+    LOG("All functions provided had at least a match");
+  }
+}
+
+bool ClientGgp::IsSelectedFunction(const FunctionInfo& func) {
```

Comment:
You are right, will update the code; thanks!

Reply:
I have updated the code. As a result, _IsSelectedFunction_ has changed aand now is called _SelectedFunctionMatch_ so it returns the matching string if found so it can be included in _capture_functions_used_by the caller.